app/uk/gov/hmrc/securemessage/controllers/SecureMessageController.scala

```diff
@@ -70,4 +70,22 @@ class SecureMessageController @Inject()(
         }
       }
   }
+
+  def getConversationContent(client: String, conversationId: String): Action[AnyContent] = Action.async {
+    implicit request =>
+      implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromHeadersAndSession(request.headers)
```

Comment:
Do we need this line ?
backendcontroller should handle it?   implicit protected def hc(implicit request: RequestHeader): HeaderCarrier =
    HeaderCarrierConverter.fromHeadersAndSessionAndRequest(request.headers, request = Some(request))

Reply:
Let me take it out and see